File: php/account.php
<?php
    class Account {
        private $id;
        private $name;
        private $document;
        private $email;
        private $password;

        public function __construct ($id, $name, $document, $email, $password) {
            $this->id = $id;
            $this->name = $name;
            $this->document = $document;
            $this->email = $email;
            $this->password = $password;
        }

        public function getId () {
            return $this->id;
        }

        public function setId ($id) {
            $this->id = $id;
        }

        public function getName () {
            return $this->name;
        }

        public function setName ($name) {
            $this->name = $name;
        }

        public function getDocument () {
            return $this->document;
        }

        public function setDocument ($document) {
            $this->document = $document;
        }

        public function getEmail () {
            return $this->email;
        }

        public function setEmail ($email) {
            $this->email = $email;
        }

        public function getPassword () {
            return $this->password;
        }

        public function setPassword ($password) {
            $this->password = $password;
        }
    }
?>

File: python/account.py
class Account:
    __id = int
    __name = str
    __document = str
    __email = str
    __password = str

    def __init__ (self, id, name, document, email, password) -> None:
        self.__id = id
        self.__name = name
        self.__document = document
        self.__email = email
        self.__password = password

    def getId (self):
        return self.__id

    def setId (self, id):
        self.__id = id

    def getName (self):
        return self.__name

    def setName (self, name):
        self.__name = name

    def getDocument (self):
        return self.__document

    def setDocument (self, document):
        self.__document = document

    def getEmail (self):
        return self.__email

    def setEmail (self, email):
        self.__email = email

    def getPassword (self):
        return self.__password

    def setPassword (self, password):
        self.__password = password